lib/src/domain/usecase/i_usecase.dart

```diff
@@ -0,0 +1,3 @@
+abstract class IUseCase<T> {
+  Future<T> repositoryCall();
```

Comment:
this could be call it 'call' only because the implementation could be to use a repository, database, etc..

Reply:
solved